Add missing dependencies to Dockerfile

filfreire opened the pull request:
## Purpose
Hello @ooPo and rest of pspdev team 👋 

This PR:
- Fixes the failing docker builds;
- Updates deprecated `MAINTAINER` tag in the dockerfile (see [this doc](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)).
- Use [pspdev](https://github.com/pspdev/pspdev) instead of [psptoolchain](https://github.com/pspdev/psptoolchain) (following PR  https://github.com/pspdev/psptoolchain/pull/160 )

## Notes

When building the dockerfile locally (e.g. running ``), I was getting the following error:

```
Cloning into 'psptoolchain'...
/build/psptoolchain /build
Couldn't find dependencies:
  - pip3
../depends/check-dependencies.sh: Failed.
The command '/bin/bash -c git clone https://github.com/pspdev/psptoolchain.git &&   pushd psptoolchain &&     ./toolchain.sh &&     popd &&   rm -Rf psptoolchain' returned a non-zero code: 1
```

I noticed the latest Github actions were also failing ([example](https://github.com/pspdev/pspdev-docker/runs/4557773234?check_suite_focus=true#step:4:1428)).

After adding `python3-pip` dependency, the build would fail with errors like `Missing GMP ...` stuff, so I compared the dependencies of the current `Dockerfile` with the ones on [prepare-debian-ubuntu.sh](https://github.com/pspdev/psptoolchain/blob/master/prepare-debian-ubuntu.sh) file of [psptoolchain](https://github.com/pspdev/psptoolchain), and indeed there were some missing:
![dependencies_from_psptoolchain](https://user-images.githubusercontent.com/11976836/153681094-56c4f60b-b297-4bc1-90ca-f515a90b7757.png)

filfreire committed:
Add missing dependencies to Dockerfile

sajattack commented:
This docker repo is an old one, the latest is here https://hub.docker.com/r/pspdev/pspdev

sajattack commented:
I think we should archive this repo.

ooPo commented:
archiving is fine by me

filfreire committed:
Use pspdev instead of psptoolchain

Update the Dockerfile taking into account https://github.com/pspdev/psptoolchain/pull/160

filfreire commented:
@ooPo @sajattack I noticed too late that this one was deprecated (after the refactor of `psptoolchain`), in the meantime I was already [converting the Dockerfile ](https://github.com/pspdev/pspdev-docker/pull/15/commits/0fa8e3d58981a620c032d12e77f08bca284b7ccd) to use `pspdev` instead 😅

I think it's better to close this PR then and you can archive the repo 🙇

sajattack commented:
@ooPo I don't have the repo permissions to do the archiving myself, could you take care of it?
